lib/api/vpn.js

```diff
@@ -0,0 +1,64 @@
+export const getClientVPNInfo = () => ({
+    method: 'get',
+    url: 'vpn',
+});
+
+export const queryVPNLogicalServerLoads = (IP) => ({
```

Comment:
You are not using `IP`.

Reply:
It appears as a parameter in the api-spec.

Reply:
ok but then it should appears in `params` right?

Reply:
Well, it's quite unclear to me where this parameter should go from the corresponding [api-spec](https://gitlab.protontech.ch/ProtonMail/Slim-API/blob/develop/api-spec/pm_api_vpn.md) . Usually such parameters appear in the `url`, but in this api-spec they don't.

I will remove the parameter (here and in other similar instances in `vpn.js`) as it's unclear from the api-spec where it should go. Let me know if I should proceed otherwise.